connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -495,6 +511,28 @@ public KafkaConsumer<byte[], byte[]> createConsumerAndSubscribeTo(Map<String, Ob
         return consumer;
     }
 
+    public KafkaProducer<byte[], byte[]> createProducer(Map<String, Object> producerProps) {
+        Map<String, Object> props = new HashMap<>(producerProps);
+
+        putIfAbsent(props, BOOTSTRAP_SERVERS_CONFIG, bootstrapServers());
+        putIfAbsent(props, ENABLE_AUTO_COMMIT_CONFIG, "false");
+        putIfAbsent(props, AUTO_OFFSET_RESET_CONFIG, "earliest");
```

Comment:
No need for `ENABLE_AUTO_COMMIT_CONFIG` or `AUTO_OFFSET_RESET_CONFIG` in the producer configs

Reply:
Good catch, removed